src/app/app-routing.module.ts

```diff
@@ -6,6 +6,7 @@ import { LoginComponent } from './login/login.component';
 import { AlbumsComponent } from './albums/albums.component';
 import { AlbumComponent } from './album/album.component';
 import { SearchResultComponent } from './search-result/search-result.component';
+import {UserSettingsComponent} from './user-settings/user-settings.component';
```

Comment:
I'm gonna be a bit boring on this , bit for me the formatting is really important : 
Turn those :
`import {UserSettingsComponent} ...`
Into : 
`import { UserSettingsComponent } ...`